provisioner/kuberpak/controllers/bundleinstance_controller.go

```diff
@@ -131,15 +131,14 @@ func (r *BundleInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 				Reason: reason,
 			})
 			return ctrl.Result{}, nil
-		} else {
-			meta.SetStatusCondition(&bi.Status.Conditions, metav1.Condition{
-				Type:    "Installed",
-				Status:  metav1.ConditionFalse,
-				Reason:  "BundleLookupFailed",
-				Message: err.Error(),
-			})
-			return ctrl.Result{}, err
 		}
+		meta.SetStatusCondition(&bi.Status.Conditions, metav1.Condition{
+			Type:    "Installed",
+			Status:  metav1.ConditionFalse,
+			Reason:  "BundleLookupFailed",
```

Comment:
unrelated: We should add these hardcoded reasons to the bundleinstance_types.go file at some point.

Reply:
Definitely, that'll particularly become nice when the E2E suite becomes non-boilerplate.